### ISSUE TITLE
S3 hosted video asset fails to generate meta

### ISSUE THREAD

**Issue by pixelmachine** [carlcs/craft-assetmetadata#14]
I'm not sure if it's the same as #4 but since that's 5 years old I figured I'd make a new issue.

I have video files that are hosted on an S3 compatible host, the videos are uploaded by users.

I'm really only after the width and height but the assets always fail to generate meta unless the `$downloadChunkSize` is equal to or above the actual size of the video. The error I get from getID3 is `unable to determine file format`.

Seems like the temporary file doesn't have enough data, I wonder if the plugin could automatically try again (a few times maybe) with larger chunk sizes if it fails first time?
